lib/std/os.zig

```diff
@@ -50,8 +50,25 @@ pub const system = if (builtin.link_libc) std.c else switch (builtin.os) {
 
 pub usingnamespace @import("os/bits.zig");
 
-/// See also `getenv`. Populated by startup code before main().
-pub var environ: [][*]u8 = undefined;
+// Using this as I don't know how to declare a different version of environ
```

Comment:
```zig
comptime {
    if (builtin.link_libc) {
        ......
    } else {
        ......
    }
}
```

Reply:
AFAIR you can't put declarations in `comptime`, I usually use something like this:

```zig
pub usingnamespace if (builtin.link_libc) struct {
//
} else {
//
};
```